app/lib/system/redis_client_pool.rb

```diff
@@ -0,0 +1,12 @@
+module System
+  module RedisClientPool
+
+    module_function
+
+    def new_pool(config = System::Application.config.redis)
```

Comment:
my suggestion is to have a method `ger` or `default` or something like this, that will memoize the object and all places would use the same object. In that way every Ruby process will use a single pool instead of using multiple pools within the process.

Can be done at a later stage as well. Something like:
```
def get
  @pool ||= new_pool
end
```

Do we need to create a different pool type at different places in porta? I think we should just use the default config and `new_pool` can even be private.